config/HSC/calibrate.py

```diff
@@ -73,8 +73,6 @@
 config.detection.isotropicGrow = True
```

Comment:
As with DECam above, this isn't adding the plugin, but rather modifying its config. I'll do some further digging on the `sfm.py` default plugins. Those might be a bit harder to remove; see `sfm.py:139` we'd have to at minimum remove items from that config list, instead of just removing lines that add new plugins as was done on this ticket.